packages/runtime/src/lib/constants.ts

```diff
@@ -152,6 +152,7 @@ TxnFields[5] = {
 
 TxnFields[6] = {
 	...TxnFields[5],
+	LastLog: null,
```

Comment:
I'm not happy with current TxnFields format(another data like this). 
We can do it better like 
```js
TxnFields = Map([
   FieldName: {
      minVersionSupport, 
      minITxnVersionSupport, 
      isArray, 
      mappingEncField: "snd"....
    }
])
```